### PR TITLE
feat : add avatar and badge components

### PR DESCRIPTION
**Pull request by viru2001**
# Added Avatar and Badge

## Description:

I created following variants in components.
 
Avatar : 
1. Image avatar
2. Text avatar

Badge :
1. avatar badge
2. text badge
3. button badge
4. icon badge

avatar-badge

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+.avatar-xs{
+    width: 5rem;
+}
+
+.avatar-text{
+    background-color: var(--primary-color);
+    color: var(--white-color);
+}
+
+.avatar-lg-text{
```
Comment (akshay-rajput) [inline components/Avatar/avatar.css]: no need for these classes

File: components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.avatar-lg{
```
Comment (akshay-rajput) [inline components/Avatar/avatar.css:1]: include height here

File: components/Badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
+/* badge on avatar */
+.badge-status {
+  width: 2rem;
+  height: 2rem;
+  border: solid 1px var(--white-color);
+}
+.badge-lg-status {
+  right: 0;
+  bottom: 1.2rem;
```
Comment (akshay-rajput) [inline components/Badge/badge.css:8]: this class is using positioning properties.. use proper name here..

File: components/Badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+}
+.status-online {
+  background-color: var(--success-color);
+}
+.badge-md-status {
+  right: 0;
+  bottom: 1rem;
+  right: 3px;
+}
+.status-offline {
```
Comment (akshay-rajput) [inline components/Badge/badge.css]: should be badge-status-offline

File: components/Badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  background-color: var(--primary-color);
+  color: var(--white-color);
+  padding: 0 7px;
+  width: 2.5rem;
+  height: 2.5rem;
+}
+.heading-badge {
+  bottom: 2rem;
+}
+.text-badge{
```
Comment (akshay-rajput) [inline components/Badge/badge.css]: name and property type mismatch

**Review by akshay-rajput — CHANGES_REQUESTED** [review]
make these minor changes

File: components/Avatar/avatar.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Mohasin-Haque) [inline components/Avatar/avatar.html:6]: Can add meta descriptions to personalise it more.
Like <meta name="author" content="Viresh" />
    <meta name="description" content="about your ui" />

File: components/Badge/badge.html [hunk 7]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Mohasin-Haque) [inline components/Badge/badge.html:6]: same can be done here also

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is clean and its good to merge but need to work on the commented part as well..

**Pull request merged**